pallets/parachain-staking/src/mock.rs

```diff
@@ -161,6 +161,8 @@ impl Default for ExtBuilder {
 			delegations: vec![],
 			collators: vec![],
 			inflation: InflationInfo {
+				blocks_per_round: MinBlocksPerRound::get(),
+				blocks_per_year: 31557600 / 12,
```

Comment:
`31557600` should be definitely moved to a constant to describe what that value means

Reply:
it's there to make moonbeam tests pass. 

we can extract it to a variable so it's clear what this is. how about this?